src/rpc/server.cpp

```diff
@@ -276,9 +276,9 @@ UniValue help(const JSONRPCRequest& jsonRequest)
         );
 
     std::string strCommand, strSubCommand;
-    if (jsonRequest.params.size() > 0)
+    if (!jsonRequest.params[0].isNull())
```

Comment:
I tend to revert this as well as this is not about optional parameters but about figuring out what type of command structure (single or sub command) is used